pkg/kubectl/cmd/get.go

```diff
@@ -35,6 +36,8 @@ import (
 	"k8s.io/kubernetes/pkg/kubectl/resource"
 	"k8s.io/kubernetes/pkg/util/i18n"
 	"k8s.io/kubernetes/pkg/util/interrupt"
+
+	"github.com/reconquest/loreley"
```

Comment:
AFAICT, this doesn't have a license. We can't merge this.

We need a verify script to look for this type of thing. It's repeatedly caused us grief.